README.md

```diff
@@ -81,6 +83,17 @@
 
 *Feel bored with the same style ? There are much more styles [here](https://github.com/harish-sethuraman/readme-components/blob/master/docs/buttonComponent.md)*
 
+# Star-Rating Component
+Showcase proficiency level of your skills with Star-Rating component
+
+_You can use the below code to generate a StarRating Component_
+
+> \_change the `&skill=` to your skill and give it a rating with `&text` param and that's all
```

Comment:
```suggestion
> change the `&skill=` to your skill and give it a rating with `&text=` param and that's all
```

Reply:
done in [91d4a7a](https://github.com/harish-sethuraman/readme-components/pull/12/commits/91d4a7ac8c868b254826f3f0531539c08280fa2f)

Reply:
Had a querie. How to update `README` to show component? Will it be displayed post deployment ?
![Screenshot 2022-03-06 at 10 33 35 PM](https://user-images.githubusercontent.com/24493084/156933662-d00795c5-50a3-489a-9caf-f3e6b870c131.png)

Reply:
Yes it will be visible post deployment. Ill merge it tomorrow in the morning